src/testRunner/unittests/evaluation/awaitUsingDeclarations.ts

```diff
@@ -1874,4 +1874,140 @@ describe("unittests:: evaluation:: awaitUsingDeclarations", () => {
             "catch",
         ]);
     });
+
+    it("deterministic collapse of Await", async () => {
+        const { main, output } = evaluator.evaluateTypeScript(

```

Comment:
Are these tests of existing functionality? I don't see how changing the type of [Async]IteratorObject would change its runtime behaviour.

Reply:
This test was in the wrong test suite. I merely moved while adding other tests.